postman/README.md

```diff
@@ -0,0 +1,37 @@
+# Postman code examples
+
+This directory contains examples for use with Postman
+
+Script | Link |
+--- | --- |
+Create and Schedule Broadcast|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/5a18ec66a11819ae4a3f)
+Find Subscribers Across Lists|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/f6ca23098db76a87c5ad)
+Get Broadcast Details|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/94304813187cfa46ab97)
+Get Broadcast Open Click|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/31f187b357a84c3a2ca8)
+Get Broadcast Stats|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/007cb5e575b093b53444)
+Get Clicks|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/3ef21f19bcf8bc8d7504)
+Get Integrations|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8f521e5bd78c83b9e8e8)
+Get Tags|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2e2c3ee4ff07d60146db)
+Get Webforms Detail|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2e2c3ee4ff07d60146db)
+Manage Custom Fields|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/46ea48d4f90e6595d421)
+Manage Subscriber|
+Move Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/37ffdf13b6588329b6da)
+Please follow this tutorial to import this [Postman Collection](https://www.getpostman.com/docs/collections).
+
+## Keys used in APIs that need to be created by the user:
+
+### For all requests
+
+- consumer_key
+- consumer_secret
+- access_token
+- access_secret
```

Comment:
All the collections use `token_secret` not `access_secret`

Reply:
Updated